Test/git-issues/git-issue-854.dfy.expect

```diff
@@ -0,0 +1,3 @@
+git-issue-854.dfy(10,32): Error: expression is not allowed to invoke a lemma (l)
+git-issue-854.dfy(10,33): Error: lemma call is not allowed to be used in an expression context (l)
```

Comment:
Mentioning "lemma" instead of "method" is an improvement. Both messages are still a bit confusing, since there _is_ a way to calls lemmas from expressions. When pertaining to a lemma (not a general method), the first message could be further improved by appending "(unless it precedes another expression)". But the trouble with that message here is that there _is_ another expression that _seems_ to follow the lemma call--the parser picks the semi-colon to go with the `assert`, though, but that will probably go unnoticed to someone who is confused by this in the first place. So, I don't have any better suggestions. :(

Reply:
Left as is.